console/src/components/Services/ApiExplorer/Actions.js

```diff
@@ -163,15 +177,15 @@ const createWsClient = (url, headers) => {
   const websocketProtocol = gqlUrl.protocol === 'https:' ? 'wss' : 'ws';
   const headersFinal = getHeadersAsJSON(headers);
   const graphqlUrl = `${websocketProtocol}://${url.split('//')[1]}`;
-  const client = new SubscriptionClient(graphqlUrl, {
-    connectionParams: {
-      headers: {
-        ...headersFinal,
-      },
-    },
-    reconnect: true,
-  });
-  return client;
+
+  if (!WEBSOCKET_SUBSCRIPTION_CLIENT) {
```

Comment:
Tried using `instanceof` against the class `SubscriptionClient`, didn't work as expected.